lldb/test/API/functionalities/tsan/swift/Makefile

```diff
@@ -1,6 +1,9 @@
 
 SWIFT_SOURCES := main.swift
 
+TSAN_DYLIB_DIR := $(CLANG_MODULE_CACHE_DIR)/../../../lib/lldb/swift/clang/lib/darwin/
+LD_EXTRAS := -lclang_rt.tsan_osx_dynamic -L$(TSAN_DYLIB_DIR)
```

Comment:
Should we modify `Makefile.rules` such that all tests use the just-built `compiler-rt`?

Reply:
I think this is a good idea, but we should probably make sure we're consistent and confident about finding the right directory.